test/lib/import/brca/providers/st_george_old/st_george_handler_old_test.rb

```diff
@@ -10,6 +10,22 @@ def setup
     @logger = Import::Log.get_logger
   end
 
+  test 'process_fields' do
+    e_batch = EBatch.create(original_filename: 'test_file',
```

Comment:
I'm assuming this `e_batch` isn't persisted, it'd be invalid with `RJ7_2`. `e_batch.reload` shows that it is persisted as part of `handler.finalize`.

Reply:
Its because , we change the e_batch properties via handler using instance variable, So any change made there is reflected in e_batch as well and that gets persisted in database.. EBatch.last.provider also gives same thing.